Language/Libraries_and_Scripts/Parts/syntax_t01.dart

```diff
@@ -7,24 +7,31 @@
 /// part directives.
 /// A part directive specifies a URI where a Dart compilation unit that should
 /// be incorporated into the current library may be found.
-/// partDirective:
-///   metadata part uri ';'
-/// ;
-/// partHeader:
-///   metadata part of identifier (‘.’ identifier)* ‘;’
-/// ;
-/// partDeclaration:
-///   partHeader topLevelDefinition* EOF
-/// ;
-/// A part header begins with part of followed by the name of the library the
-/// part belongs to. A part declaration consists of a part header followed by a
-/// sequence of top-level declarations.
-/// @description Checks that part unit can not contain an import directive.
+///
+/// <partDirective> ::= <metadata> `part' <configurableUri> `;'
+/// <partHeader> ::= <metadata> `part' `of' <uri> `;'
+/// <partDeclaration> ::=
+///   <partHeader> <importOrExport>* <partDirective>* (<metadata>
+///   <topLevelDeclaration>)* <EOF>
+///
+/// @description Checks that `partDirective`, `partHeader` and `libraryName` may
+/// contain metadata.
```

Comment:
I don't see anything like `@Annot1() library;`, is there a case where `libraryName` metadata is tested? Similarly for `part of` in the parts.

Reply:
Indeed. Thank you. Fixed.